libr/bin/bfile.c

```diff
@@ -885,6 +878,24 @@ static int is_data_section(RBinFile *a, RBinSection *s) {
 	return strstr (s->name, "_const") != NULL;
 }
 
+// XXX : This is O(n) for strings, we need to query strings by offset in the rbinobj directly
+static RBinString *find_string_at (RBinObject *bo, RList *ret, ut64 addr) {
+	if (addr != 0 && addr != UT64_MAX) {
+#if 1
```

Comment:
remember to clean this before merging (after travis is green :)

Reply:
yep. its there for benchmarking reasons :)

Reply:
i will kill this ifdef when this pr gets green